consensus/src/chained_bft/consensusdb/schema/block/mod.rs

```diff
@@ -37,13 +37,41 @@ impl<T: Payload> KeyCodec<BlockSchema<T>> for HashValue {
     }
 }
 
-impl<T: Payload> ValueCodec<BlockSchema<T>> for Block<T> {
+#[derive(Clone)]
+/// SchemaBlock is a crate wrapper for Block that is defined outside this crate.
```

Comment:
What is wrong with defining `ValueCodec` for `Block` in the new location of the consensus crate?

Reply:
That's doable and a reasonable alternative though it does affect the code readability of the schema.rs by moving that over.